src/coreclr/jit/morph.cpp

```diff
@@ -2897,7 +2900,11 @@ void Compiler::fgInitArgInfo(GenTreeCall* call)
     // Compute the maximum number of arguments that can be passed in registers.
     // For X86 we handle the varargs and unmanaged calling conventions
 
+#ifdef UNIX_X86_ABI
+    if (call->IsVarargs())
```

Comment:
VarArgs is not supported on Unix, so this will be always false. Would it be better to put this whole block under `#ifndef UNIX_X86_ABI` ?